crates/forge/bin/cmd/inspect.rs

```diff
@@ -29,18 +30,14 @@ pub struct InspectArgs {
     #[arg(value_enum)]
     pub field: ContractArtifactField,
 
-    /// Pretty print the selected field, if supported.
-    #[arg(long)]
-    pub pretty: bool,
```

Comment:
we should keep this with a warning or document as a breaking change

Reply:
yeah, will be marked in release notes as breaking change given the breaking label